packages/fxa-content-server/app/scripts/lib/fxa-client.js

```diff
@@ -609,8 +609,13 @@ FxaClientWrapper.prototype = {
         sessionToken: true,
       };
 
-      return client
-        .passwordForgotVerifyCode(code, token, {})
+      return Promise.resolve()
+        .then(() => {
+          if (options.accountResetToken) {
+            return { accountResetToken: options.accountResetToken };
+          }
+          return client.passwordForgotVerifyCode(code, token, {});
```

Comment:
Looks like one of these paths returns a Promise and the other returns an Object. This ok?

Reply:
@jodyheavener Yea this should be ok since its all in the promise chain.